common/mariadb/scripts/mariadb-rename-check-constraints.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+# shellcheck shell=bash
+# shellcheck disable=SC1091
+# shellcheck disable=SC2250
+# shellcheck disable=SC2321
+set -o pipefail
+set -u
+
+source /usr/bin/common-functions.sh
+
+MYSQL_ADDRESS=${MYSQL_ADDRESS:-""}
+MYSQL_USER=${MYSQL_USER:-root}
+MYSQL_PASSWORD=${MYSQL_PASSWORD:-""}
+
+if [[ -z "${MYSQL_USER}" ]] || [[ -z "${MYSQL_ADDRESS}" ]] || [[ -z "${MYSQL_PASSWORD}" ]]; then
+    logerror "check_env" "Required environment variables are not set (MYSQL_USER, MYSQL_ADDRESS, MYSQL_PASSWORD)"
+    exit 1
+fi
+
+export MYSQL_PWD="${MYSQL_PASSWORD}"
```

Comment:
cannot find where this is used.

Reply:
This env is being used by `mariadb-client`, https://mariadb.com/kb/en/mariadb-environment-variables/